### ISSUE TITLE
MARIADB_EXTRA_FLAGS is not working

### ISSUE THREAD

**Issue by s-mansouri** [bitnami/bitnami-docker-mariadb#194]
**Description**
Hi,
I put MARIADB_EXTRA_FLAGS as environment variables but the command-line flags not affected
I use an image with version '10.1.36-debian-9-r54' and I realized that after version '10.1.35-debian-9-r14' this variable removed but it still exists in documents
 
**Steps to reproduce the issue:**
```yml
version: '3.6'
services:
  mariadb-master:
    image: 'bitnami/mariadb:10.1.36-debian-9-r54'
    environment:
      - MARIADB_REPLICATION_MODE=master
      - MARIADB_REPLICATION_USER=repl_user
      - MARIADB_USER=my_user
      - MARIADB_DATABASE=my_database
      - ALLOW_EMPTY_PASSWORD=yes
      - MARIADB_EXTRA_FLAGS=--server-id=3000
      - MARIADB_ROOT_PASSWORD=my_root_password
  mariadb-slave:
    image: 'bitnami/mariadb:10.1.36-debian-9-r54'
    depends_on:
      - mariadb-master
    environment:
      - MARIADB_REPLICATION_MODE=slave
      - MARIADB_REPLICATION_USER=repl_user
      - MARIADB_USER=my_user
      - MARIADB_DATABASE=my_database
      - MARIADB_MASTER_HOST=mariadb-master
      - MARIADB_MASTER_PORT_NUMBER=3306
      - MARIADB_MASTER_ROOT_PASSWORD=my_root_password
      - MARIADB_EXTRA_FLAGS=--server-id=3001
      - ALLOW_EMPTY_PASSWORD=yes
```
**Describe the results you received:**

MariaDB [my_database]> SELECT @@GLOBAL.server_id;
+---------------------------------+
| @@GLOBAL.server_id   |
+---------------------------------+
|                652                   |
+---------------------------------+
1 row in set (0.00 sec)

**Describe the results you expected:**

I expected to get server_id = 3000

**Version**

docker Version:  18.09.5

**Comment by alemorcuq** [comment]
Hi, @s-mansouri.

Thank you for reporting this. It looks like we are setting the `--server-id` flag to a random number despite what the user specifies:

https://github.com/bitnami/bitnami-docker-mariadb/blob/c4a606c82a433b132dd00d22e091b72b527db300/10.1/debian-9/rootfs/libmysql.sh#L224

We will fix it.

Regards,
Alejandro

**Comment by alemorcuq** [comment]
Hi, @s-mansouri.

We've released a new set of containers that should fix your issue. They are the following:

```
10.1.41-ol-7-r35
10.3.17-ol-7-r35
10.2.26-ol-7-r35
10.3.17-debian-9-r33
10.2.26-debian-9-r35
10.1.41-debian-9-r35
```

Thank you again for bringing this to our attention.

Regards,
Alejandro

**Comment by s-mansouri** [comment]
Thank you for your support. But this isn't still working. if I set MARIADB_EXTRA_FLAGS container stock in this step:
```05:56:51.67 INFO  ==> Initializing mariadb database...```

If I set EXTRA_FLAGS, which I found in the merged pull request. server-id doesn't set and it is still a random number:

```yml
version: '3.6'
services:
  mariadb-master:
    image: 'bitnami/mariadb:10.1.41-debian-9-r35'
    ports:
      - '3306'
    volumes:
      - ./initscript:/docker-entrypoint-initdb.d
    environment:
      - MARIADB_REPLICATION_MODE=master
      - MARIADB_REPLICATION_USER=repl_user
      - MARIADB_USER=my_user
      - MARIADB_DATABASE=my_database
      # ALLOW_EMPTY_PASSWORD is recommended only for development.
      - ALLOW_EMPTY_PASSWORD=yes
      - MARIADB_ROOT_PASSWORD=my_root_password
      - MARIADB_EXTRA_FLAGS=3000
```
log: 
```bash
 05:53:49.27 
 05:53:49.27 Welcome to the Bitnami mariadb container
 05:53:49.28 Subscribe to project updates by watching https://github.com/bitnami/bitnami-docker-mariadb
 05:53:49.28 Submit issues and feature requests at https://github.com/bitnami/bitnami-docker-mariadb/issues
 05:53:49.28 Send us your feedback at containers@bitnami.com
 05:53:49.28 
 05:53:49.28 INFO  ==> ** Starting MariaDB setup **
 05:53:49.30 INFO  ==> Validating settings in MYSQL_*/MARIADB_* env vars..
 05:53:49.30 WARN  ==> You set the environment variable ALLOW_EMPTY_PASSWORD=yes. For safety reasons, do not use this flag in a production environment.
 05:53:49.31 INFO  ==> Initializing mariadb database...
```
with EXTRA_FLAGS:
``` yaml
version: '3.6'
services:
  mariadb-master:
    image: 'bitnami/mariadb:10.1.41-debian-9-r35'
    ports:
      - '3306'
    volumes:
      - ./initscript:/docker-entrypoint-initdb.d
    environment:
      - MARIADB_REPLICATION_MODE=master
      - MARIADB_REPLICATION_USER=repl_user
      - MARIADB_USER=my_user
      - MARIADB_DATABASE=my_database
      # ALLOW_EMPTY_PASSWORD is recommended only for development.
      - ALLOW_EMPTY_PASSWORD=yes
      - MARIADB_ROOT_PASSWORD=my_root_password
      - EXTRA_FLAGS=3000
```
Output:
```sql

MariaDB [my_database]> SELECT @@GLOBAL.server_id;
+--------------------+
| @@GLOBAL.server_id |
+--------------------+
|                974 |
+--------------------+
1 row in set (0.00 sec)

```

**Comment by s-mansouri** [comment]
Thank you for your support.
Is it possible to fix this for version 10.1.36?

**Comment by alemorcuq** [comment]
> @alemorcuq Thank you for your support.
> Is it possible to fix this for version 10.1.36?

Hi, @s-mansouri.

I'm afraid we can't do it, you will have to create your own container with the changes we did to `libmysql.sh`.

Regards,
Alejandro
